tbx/blog/models.py

```diff
@@ -216,7 +221,7 @@ def blog_index(self):
             # just return first blog index in database
             return BlogIndexPage.objects.first()
 
-    @property
+    @cached_property
     def first_author(self):
         """Safely return the first author if one exists."""
         author = self.authors.first()
```

Comment:
Andy mentioned that he thought that first() might make an extra db query regardless of the `@cached_property` here (although he also said this is still a good performance improvement)

Reply:
Indeed. It will do so on the first use, but we have several templates like blog_detail.html which uses this multiple times, so that helps there

Reply:
Specifically, I was worried that we were prefetching the authors themselves and not really benefiting from it here. I need to take another look at `first()` under the hood, to see how it interacts with prefetched values.

Reply:
Ah yes, looks like we should be good: https://github.com/django/django/blob/main/django/db/models/query.py#L433-L434